docs/concepts/components.md

```diff
@@ -53,8 +53,8 @@ component for automatic management of DNS entries for Ingress resources.
 ### Node management
 
 Lokomotive provides the
-[calico-hostendpoint-controller](../configuration-reference/components/calico-hostendpoint-controller.md)
-component for adding and removing [Calico HostEndpoint
+[calico-hostendpoint-controller](https://github.com/kinvolk/calico-hostendpoint-controller/)
```

Comment:
Following this change, the controller is no longer a Lokomotive component as it is deployed as part of the control plane bootstrap and cannot be controlled via `lokoctl`. Therefore, I would remove this mention here and consider mentioning the controller in a *concepts* document which explains how Lokomotive is deployed on Packet (a document which we don't have at the moment).

Reply:
done